Frontend/src/components/MentorProfileModal.jsx

```diff
@@ -24,38 +24,59 @@ function MentorProfileModal ({handleCheckboxChange, handleDropdownToggle, dropdo
         bio: '',
         skills: selectedSkills.join(', '),
     })
+
+    useEffect(() => {
+        if (mentorData) {
+            setFormData({
```

Comment:
Did you double check this is the correct way to default on null (assuming that's what your attempting to do). I believe it should be ?? not || 
I think the current implementation will leave all these values as booleans

Reply:
https://stackoverflow.com/questions/61480993/when-should-i-use-nullish-coalescing-vs-logical-or

This explains the difference but I think both shoudl be ok?

Reply:
What I have put seems to be ok, so I think I may just keep it.